models/curvefi/curvefi_trades.sql

```diff
@@ -2,10 +2,10 @@
         tags = ['dunesql'],
         partition_by = ['block_month'],
         alias = alias('trades'),
-        post_hook='{{ expose_spells(\'["ethereum","avalanche_c","optimism","fantom"]\',
+        post_hook='{{ expose_spells(\'["ethereum","avalanche_c","optimism","fantom","celo"]\',
```

Comment:
you can leave this for now, as it's existing spell, we can consider on our future cleanup